web/html/src/manager/admin/task-engine-status/taskotop.tsx

```diff
@@ -0,0 +1,268 @@
+    var statusValues = { running: 0, ready_to_run: 1, failed: 2, interrupted: 3, skipped: 4, finished: 5 };
+    var a = statusValues[aRaw[columnKey]];
+    var b = statusValues[bRaw[columnKey]];
+    var result = (a > b ? 1 : a < b ? -1 : 0) || this.sortByEndTime(aRaw, bRaw, "endTime", sortDirection);
+    return (result || Utils.sortById(aRaw, bRaw)) * sortDirection;
+  };
+
+  sortByNumber = (aRaw, bRaw, columnKey, sortDirection) => {
+    var a = aRaw[columnKey];
+    var b = bRaw[columnKey];
+    var result = a > b ? 1 : a < b ? -1 : 0;
+    return (result || Utils.sortById(aRaw, bRaw)) * sortDirection;
+  };
+
+  searchData = (datum, criteria) => {
+    if (criteria) {
+      return datum.name.toLowerCase().includes(criteria.toLowerCase());
+    }
+    return true;
+  };
+
+  decodeStatus = status => {
+    var cell;
+    switch (status) {
+      case "running":
+        cell = (
+          <div>
+            <i className="fa fa-cog fa-spin"></i>
+            {t(" running")}
+          </div>
+        );
+        break;
+      case "finished":
+        cell = (
+          <div className="text-success">
+            <i className="fa fa-thumbs-o-up"></i>
+            {t(" finished")}
+          </div>
+        );
+        break;
+      case "failed":
+        cell = (
+          <div className="text-danger">
+            <i className="fa fa-exclamation-triangle"></i>
+            {t(" failed")}
+          </div>
+        );
+        break;
+      case "interrupted":
+        cell = (
+          <div className="text-warning">
+            <i className="fa fa-stop"></i>
+            {t(" interrupted")}
+          </div>
+        );
+        break;
+      case "ready_to_run":
+        cell = (
+          <div className="text-primary">
+            <i className="fa fa-list-ul"></i>
+            {t(" ready to run")}
+          </div>
+        );
+        break;
+      case "skipped":
+        cell = (
+          <div className="text-muted">
+            <i className="fa fa-angle-double-right"></i>
+            {t(" skipped")}
+          </div>
+        );
+        break;
+      default:
+        cell = null;
+    }
+    return cell;
+  };
+
+  buildRows = jobs => {
+    return Object.keys(jobs).map(id => jobs[id]);
+  };
+
+  render() {
+    const data = this.state.serverData;
+    const title = (
+      <div className="spacewalk-toolbar-h1">
+        <h1>
+          <i className="fa fa-tachometer"></i>
+          {t("Task Engine Status")}
+          <HelpLink url="reference/admin/task-engine-status.html" />
+        </h1>
+      </div>
+    );
+    const headerTabs = (
+      <div className="spacewalk-content-nav">
+        <ul className="nav nav-tabs">
+          <li>
+            <a className="js-spa" href="/rhn/admin/TaskStatus.do">
+              {t("Last Execution Times")}
+            </a>
+          </li>
+          <li className="active js-spa">
+            <a href="/rhn/manager/admin/runtime-status">{t("Runtime Status")}</a>
```

Comment:
This link had an empty href (`href=""`) in the original source, based on other uses I _think_ this should be the right URL.

Reply:
Yes, it is the same link of the current page actually, so it makes sense to fill with it. Thanks for the catch